### ISSUE TITLE
Consider vertical alignment in system-by-system output

### ISSUE THREAD

**Issue by uliska** [jperon/lyluatex#76]
When we have `insert=systems` the vertical alignment should be handled differently when the system happens to be at the top or bottom of a page.

Currently to top edge of the PDF will be at the top edge of the type area, but actually it should be handled just like the margins for fullpage scores.

**Comment by jperon** [comment]
I doubt it's really possible to know whether we are at the top or (even more) at the bottom of the page. IMHO this shouldn't be a blocking issue.

**Comment by uliska** [comment]
Yes, it is a difficult situation. And there *are* some inconsistencies that might not be fixable at all: for example we don't have any way to control horizontal protrusion in fullpage scores: there we have margins and line-width, and essentially unlimited protrusion (so there it's clearly the user's responsibility to avoid problems).

I have removed the milestone but suggest to keep the issue open.
